python/vowpalwabbit/pyvw.py

```diff
@@ -562,11 +562,16 @@ def __init__(self, costs=[], prediction=0):
     def from_example(self, ex):
         class wclass:
             def __init__(self,
-                         label,
-                         cost=0.,
-                         partial_prediction=0.,
-                         wap_value=0.):
-                self.label = label
+                        action=None,
```

Comment:
Cost sensitive labels do not call this term an action - these are classes. I think for this class though you should revert the label ->action change to reduce churn/scope of this PR

Reply:
Sure, I will remove that change